definitions/features/pulpcore.rb

```diff
@@ -3,10 +3,34 @@
 class Features::Pulpcore < ForemanMaintain::Feature
   include ForemanMaintain::Concerns::PulpCommon
 
+  TIMEOUT_FOR_TASKS_STATUS = 300
+  RETRY_INTERVAL_FOR_TASKS_STATE = 10
```

Comment:
Do we expect these two to change or need to be changed by a user?

Reply:
The timeout yes, that's what the question in https://github.com/theforeman/foreman_maintain/pull/856#discussion_r1639682187 was about.
The retry interval probably not?